set-version-env.js

```diff
@@ -0,0 +1,13 @@
+import { readFileSync } from 'fs'
+import { fileURLToPath } from 'url'
+import { dirname, join } from 'path'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+
+const packageJsonPath = join(__dirname, 'package.json')
+const packageJson = JSON.parse(readFileSync(packageJsonPath, 'utf-8'))
+
+const { version } = packageJson
+
+console.log(`VERSION=${version}`)
```

Comment:
It is missing the `process.env.VERSION=version` - I will add it. The idea is to use this script to read the version from package json and set it into the `VERSION` env var instead of importing the `package.json` file directly in the middleware.js - which was causing several failures in the tests because it couldn't resolve the `.json` import. The version is used by the middleware that returns the API version.

Reply:
Oh ok got it - because new tests are running against actual source code not esbuild bundle.

Please next time just leave a comment in the PR description to explain! 🙏